geoportailv3/static/js/wmsservice.js

```diff
@@ -468,6 +483,11 @@ app.WmsHelper.prototype.createWmsLayers = function(map, layer) {
       source: new ol.source.ImageWMS(imgOptions)
     });
   }
+  if (has3857) {
+    newLayer.getSource().set('olcs.projection', ol.proj.get('EPSG:3857'));
+  } else if (hasWGS84) {
+    newLayer.getSource().set('olcs.projection', ol.proj.get('EPSG:4326'));
```

Comment:
Please check first if 4326 is supported.

Reply:
by what ?

Reply:
OK, I see in Jira it is a request from the client.